Makefile

```diff
@@ -25,17 +23,17 @@ setup: ## Configures your Azure ACR with sandbox credentials for testing purpose
 	az account set --subscription ${ACR_SUBSCRIPTION}
 	az configure --defaults acr=${ACR}
 	az acr helm repo add
-	az aks get-credentials --resource-group ${AKS_RESOURCE_GROUP} --name ${AKS_CLUSTER}
+	az aks get-credentials --overwrite-existing --resource-group ${AKS_RESOURCE_GROUP} --name ${AKS_CLUSTER} 
 
 clean: ## Removes the installed chart
 	-helm delete --purge ${RELEASE}
-	-kubectl delete pod ${TEST} -n ${NAMESPACE}
+	-kubectl delete --namespace ${NAMESPACE} pod/${TEST}
```

Comment:
you're deleting the whole chart-tests namespace here on the cluster the pipeline uses for testing, probably not the best idea, what was wrong with what was there previously

Reply:
Actually it just deletes the pod ...  hence pod/
I modified it when I was looking for the reason it was not actually deleting the pod.
It was the name of the char that actually did it.
This is works just fine.